po/de.po

```diff
@@ -24811,6 +25145,106 @@ msgstr "Lasse %s mit Backup-Suffix '%s' aus.\n"
 msgid "Do you really want to send %s? [y|N]: "
 msgstr "Wollen Sie %s wirklich versenden? [y|N]: "
 
+#~ msgid "Stage mode change [y,n,a,q,d%s,?]? "
+#~ msgstr "Modusänderung der Staging-Area hinzufügen [y,n,a,q,d%s,?]? "
+
+#~ msgid "Stage deletion [y,n,a,q,d%s,?]? "
+#~ msgstr "Löschung der Staging-Area hinzufügen [y,n,a,q,d%s,?]? "
+
+#~ msgid "Stage this hunk [y,n,a,q,d%s,?]? "
+#~ msgstr "Diesen Patch-Block der Staging-Area hinzufügen [y,n,a,q,d%s,?]? "
+
+#~ msgid ""
+#~ "If the patch applies cleanly, the edited hunk will immediately be\n"
+#~ "marked for staging.\n"
+#~ msgstr ""
+#~ "Wenn der Patch sauber angewendet werden kann, wird der bearbeitete Patch-"
+#~ "Block\n"
+#~ "direkt als zum Hinzufügen zur Staging-Area markiert.\n"
+
+#~ msgid ""
+#~ "y - stage this hunk\n"
+#~ "n - do not stage this hunk\n"
+#~ "q - quit; do not stage this hunk or any of the remaining ones\n"
+#~ "a - stage this and all the remaining hunks\n"
+#~ "d - do not stage this hunk nor any of the remaining hunks\n"
+#~ msgstr ""
+#~ "y - diesen Patch-Block zum Commit vormerken\n"
+#~ "n - diesen Patch-Block nicht zum Commit vormerken\n"
+#~ "q - Beenden; diesen oder alle verbleibenden Patch-Blöcke nicht zum Commit "
+#~ "vormerken\n"
+#~ "a - diesen und alle weiteren Patch-Blöcke zum Commit vormerken\n"
+#~ "d - diesen und alle weiteren Patch-Blöcke nicht zum Commit vormerken\n"
+
+#~ msgid "could not copy '%s' to '%s'."
+#~ msgstr "Konnte '%s' nicht nach '%s' kopieren."
+
+#~ msgid "malformed ident line"
+#~ msgstr "Fehlerhafte Identifikationszeile."
+
+#~ msgid "corrupted author without date information"
+#~ msgstr "Fehlerhafter Autor ohne Datumsinformationen."
+
+#~ msgid "could not parse '%.*s'"
+#~ msgstr "Konnte '%.*s' nicht parsen."
+
+#~ msgid "could not checkout %s"
+#~ msgstr "Konnte %s nicht auschecken."
+
+#~ msgid "filename in tree entry contains backslash: '%s'"
+#~ msgstr "Dateiname im Tree-Eintrag enthält Backslash: '%s'"
+
+#~ msgid "Use -f if you really want to add them.\n"
+#~ msgstr "Verwenden Sie -f wenn Sie diese wirklich hinzufügen möchten.\n"
```

Comment:
Komma nach -f, denke ich.

Reply:
Ja, das stimmt. Ist eine auskommentierte Übersetzung, aber ändere ich trotzdem ;-)
Glaube mal, dass diese Übersetzungen wieder jederzeit aktiviert werden könnten.. (@ralfth ?)

Reply:
Die auskommentierten Änderungen (die wohl durch das Update auskommentiert werden) bitte unbedingt entfernen. Es gab in der Vergangenheit Probleme damit. Danke.